src/LykkePublicAPI/Controllers/MarketController.cs

```diff
@@ -41,6 +41,44 @@ public class MarketController : Controller
         /// <returns></returns>
         [HttpGet]
         public async Task<IEnumerable<ApiMarketData>> Get()
+        {
+            return await _cache.TryGetFromCacheAsync(
+                "Market:Overall", 
```

Comment:
here maybe better :Market:Overall
need check name this key in radis.... i thisk this name = FinanceDataCacheInstanceMarket:Overall without split FinanceDataCacheInstance and Market
But it's not 100% :)

Reply:
Ending ":" is added here - https://github.com/LykkeCity/LykkePublicAPI/pull/32/files#diff-1f35f465ab1d21238e737207d374c102R107